app/components/preprint-form-authors.js

```diff
@@ -10,26 +10,158 @@ export default CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
     permissionChanges: {},
     bibliographicChanges: {},
     valid: Ember.computed.alias('newContributorId'),
+    permissionToggle: false,
+    bibliographicToggle: false,
```

Comment:
The reason I exposed this was because even if bibliographic info is hidden, the bibliographic rules still apply - enforced on the model.  So for example, you might not be able to remove a contributor because doing so would leave no bibliographic contributors.   But we wouldn't be even telling them that bibliographic contributors part is a constraint.  

So then you would be prevented from making a request here, but no way to change it from this view, or explanation of why this is happening. However users could manipulate this information via the OSF.